code/datums/gamemode/dynamic/dynamic_rulesets_latejoin.dm

```diff
@@ -154,7 +154,7 @@
 	role_category = /datum/role/revolutionary
 	restricted_from_jobs = list("Merchant","AI", "Cyborg", "Mobile MMI", "Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Head of Personnel", "Chief Engineer", "Chief Medical Officer", "Research Director", "Internal Affairs Agent")
 	enemy_jobs = list("AI", "Cyborg", "Security Officer","Detective","Head of Security", "Captain", "Warden")
-	required_enemies = list(2,2,1,1,1,1,1,0,0,0)
+	required_pop = list(20,20,15,15,15,15,15,0,0,0)
```

Comment:
Revolutionaries can roll on deadpop but given what "requirements" has (can roll revs at 10-14 players) it's not THAT much of a problem

Reply:
provocateur is a single rev dude.